packages/common/reader/__tests__/fixtures/test-doc.ts

```diff
@@ -0,0 +1,2 @@
+export const snapshot =
```

Comment:
rename to `__fixtures__`

Reply:
save them as bin files, large utf8 content will make git slower